server/src/config/database.config.ts

```diff
@@ -2,7 +2,7 @@ import { TypeOrmModuleOptions } from '@nestjs/typeorm';
 
 export const databaseConfig: TypeOrmModuleOptions = {
   type: 'postgres',
-  host: 'immich_postgres',
+  host: process.env.HOST || 'immich_postgres',
```

Comment:
Let's change the `process.env.HOST` to `process.env.DB_HOSTNAME`

Reply:
And also same weird thing happened here with the "ignored" `Rename database host env variable to DB_HOST]` 🤔 . Seems like a "GitHub bug" to me 😅 .
I wonder, maybe if I force push an empty commit in `add/albums` it would "fix it".

Reply:
The forced push in `add/albums` did the trick 🎉